src/re_frame/loggers.cljc

```diff
@@ -0,0 +1,36 @@
+(ns re-frame.loggers
+  (:require
+    [clojure.set :refer [difference]]))
+
+;; "loggers" holds the current set of logging functions.
+;; By default, re-frame uses the functions provided by js/console.
+;; Use set-loggers! to change these defaults.
+
+(def ^:private loggers
+  (atom {:log       #?(:cljs (js/console.log.bind   js/console)
+                       :clj  println)
+         :warn      #?(:cljs (js/console.warn.bind  js/console)
+                       :clj  (partial println "WARN: "))
```

Comment:
Can we use `clojure.tools.logging` here? That'll integrate with the rest of the logging world.